pkg/eval/vals/struct_map.go

```diff
@@ -7,29 +7,32 @@ import (
 	"src.elv.sh/pkg/strutil"
 )
 
-// StructMap may be implemented by a struct to mark the struct as a "struct
-// map", which causes Elvish to treat it like a read-only map. Each exported,
-// named field and getter method (a method taking no argument and returning one
-// value) becomes a field of the map, with the name mapped to dash-case.
+// The StructMap interface may be implemented by a struct to mark the struct as a psuedo-map. This
```

Comment:
Don't change this file; see my comment above about structmap vs pseudo-structmap.